### ISSUE TITLE
same version of linux on all machines

### ISSUE THREAD

**Issue by maxlevesque** [ENSchemphys/phpc-sysadmin#3]
novem and mikado : ubuntu 14.04.3 LTS server (long term support)
abalone : ubuntu 15.10 server

Since compilers, libraries etc are different with different versions of  linux, we should ease our burdden to have all machines with the same OS.

**Comment by v-kaiser** [comment]
I agree about the unification. The question is which one? I see the advantages of long term support, on the other hand, even regular Ubuntu releases are stable enough, and provide for newer versions of scientific packages.

PS: 14.04.4 LTS has come out meanwhile.
